config/test_ocp.yaml

```
@@ -11,7 +11,7 @@ defaults:
 
 logger:
   project_name: test_ocp_training
-  run_name: greedy_epochs-20_samples-100_mve
+  run_name: greedy_epochs-20_samples-100_mve_12345
```

I think you could do something like:
```suggestion
  run_name: greedy_epochs-20_samples-100_mve_${seed}
```